extensions_admin/test/unit/extensions/admin/upload/test_comps.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+import mock
+
```

Comment:
Drop this newline to group pulp with mock. Since this code is pulp_rpm, pulp is technically a separate library.